roles/apigee-hybrid-overrides/templates/overrides.yaml.j2

```diff
@@ -128,7 +128,7 @@ cassandra:
   heapNewSize: {{ overrides.cassandra.heapNewSize }}
   storage:
     capacity: {{ overrides.cassandra.storage.capacity }}
-    storageclass: {{ overrides.cassandra.storage.storageclass }}
+    storageClass: {{ overrides.cassandra.storage.storageClass }}
```

Comment:
@RiflerRick : this should be `storageclass ` 

Refer: https://cloud.google.com/apigee/docs/hybrid/v1.10/config-prop-ref

Reply:
https://cloud.google.com/apigee/docs/hybrid/v1.11/cassandra-production#configure-cassandra-storage-settings

Reply:
I have modified the vars file, its camelcase everywhere